examples/research_projects/intel_opts/textual_inversion_dfq/textual_inversion.py

```diff
@@ -982,7 +965,12 @@ def attention_fetcher(x):
             )
 
         if args.push_to_hub:
-            repo.push_to_hub(commit_message="End of training", blocking=False, auto_lfs_prune=True)
+            upload_folder(
+                repo_id=repo_id,
+                folder_path=args.output_dir,
+                commit_message="End of training",
+                ignore_patterns=["step_*", "epoch_*"],
+            )
```

Comment:
```suggestion
                ignore_patterns=["step_*", "epoch_*"],
                run_as_future=True,
            )
```

It looks like before `blocking=False` was used, meaning we would need to use `run_as_future=True` here to keep the same behavior.

Reply:
Same as before.